src/main/java/com/splunk/logging/HttpEventCollectorThrowableInfo.java

```diff
@@ -0,0 +1,145 @@
+package com.splunk.logging;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Sanitised information about a throwable that is suitable for logging. Has been decoupled from
+ * any class information in the original throwable.
+ */
+public class HttpEventCollectorThrowableInfo {
+
+    private final String message;

```

Comment:
nit: remove excessive blank lines, see the style we use in `HttpEventCollectorEventInfo.java`